src/common/schemas/profile-setting-schema.ts

```diff
@@ -0,0 +1,16 @@
+import { z } from 'zod'
```

Comment:
add TODO , for translate in zod validtion )

Reply:
done